js-miniapp-sample/src/pages/message.js

```diff
@@ -274,15 +273,17 @@ const Message = (props: MessageTypeProps) => {
           rowsMax="4"
         />
       </FormControl>
-      <FormControl className={classes.formControl}>
-        <TextField
-          id="bannerMessage"
-          label="Banner message"
-          className={classes.fields}
-          onChange={onBannerMessageChange}
-          value={message.bannerMessage}
-        />
-      </FormControl>
+      {message.id !== 2 && (
```

Comment:
We don't need to do this right now: but I think we should create constants for these ids so that this is easier to read.

Reply:
yeah you're right